CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased] - 2019-07-12
+
+- Add Rails 6 support
+- Lock gems for Rails 5 specs
+
 ## [Unreleased] - 2017-12-21
 
 ## Fixed
```

Comment:
Oops, I just noticed my mistake here, it should be a third-level header.

```suggestion
### Fixed
```

Reply:
And same thing in the 1.15.1. section that I can't reach in this PR review, but if you feel like fixing it, please go ahead!